eth/eventhandler/event_handler.go

```diff
@@ -116,8 +117,11 @@ func (eh *EventHandler) HandleBlockEventsStream(logs <-chan executionclient.Bloc
 		if err != nil {
 			return 0, fmt.Errorf("failed to process block events: %w", err)
 		}
-
 		lastProcessedBlock = blockLogs.BlockNumber
+
+		if lastProcessedBlock <= math.MaxInt64 {
+			lastProcessedBlockGauge.Record(ctx, int64(lastProcessedBlock))
+		}
```

Comment:
It seems converting `uint64` -> `int64` is a common thing we do multiple times, perhaps we should have a helper function for this instead to simplify away all these additional `if`s

Reply:
I agree.
[Wdyt](https://github.com/ssvlabs/ssv/pull/1955)?

Reply:
Yes that's probably the best way to do it.